monors.py

```diff
@@ -487,16 +486,71 @@ def main():
 
     gh_slack_usermapping = json.load(open("monors_slack_users.json")) if slack else None
 
-    reviewers = sorted([collaborator ["login"] for collaborator in get_collaborators (gh, cfg["owner"], cfg["repo"])])
+    headers = { 'AUTHORIZATION': "token %s" % cfg ["token"], 'ACCEPT': 'application/json', 'CONTENT-TYPE': 'application/json' }
+    query = {}
+    query["query"] = '''
+            {
+                rateLimit {
+                    limit
+                    cost
+                    remaining
+                    resetAt
+                }
+                repository(owner: "mono", name: "mono") {
+                    collaborators(first: 100) {
+                        edges { node { login } }
+                    }
+                    pullRequests(last: 30, states: OPEN, orderBy: {field: UPDATED_AT, direction: ASC}) {
```

Comment:
same here. we have almost 200 open PRs right now on mono/mono.

Reply:
We requested the last 50 PRs in the old version, I think I'll change to that. I think the reasoning was that the automerge command is most likely on a recent PR, not one from two years ago.